src/components/EmptyWishlist/EmptyWishlist.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export const EmptyWishlist = () => (
+  <div className="empty-wishlist-area">
+    <h1 className="empty-wishlist-heading">Your Wishlist is empty!</h1>
+    <p className="empty-wishlist-msg1">
+      Seems like you don't have a wish here.
+    </p>
+    <p className="empty-wishlist-msg2">Make a wish!</p>
```

Comment:
I think instead of creating separate classes `empty-wishlist-msg1` and  `empty-wishlist-msg2` , this functionality can be merged into a single class. Both these are same **empty cart message** component.

Reply:
yes will change it